schema/generator.go

```diff
@@ -199,6 +206,28 @@ func (g *Generator) generateDDLs(desiredDDLs []DDL) ([]string, error) {
 	return ddls, nil
 }
 
+func (g *Generator) removeColumnConstraints(currentTable *Table, columnName string, ddls []string) []string {
+	for _, column := range currentTable.columns {
+		if column.name == columnName && column.defaultDef != nil && column.defaultDef.name != "" {
+			ddl := fmt.Sprintf("ALTER TABLE %s DROP CONSTRAINT %s", g.escapeTableName(currentTable.name), g.escapeSQLName(column.defaultDef.name))
+			ddls = append(ddls, ddl)
+		}
+	}
+
+	for _, index := range currentTable.indexes {
+		if index.primary {
+			for _, indexColumn := range index.columns {
+				if indexColumn.column == columnName {
+					ddl := fmt.Sprintf("ALTER TABLE %s DROP CONSTRAINT %s", g.escapeTableName(currentTable.name), g.escapeSQLName(index.name))
+					ddls = append(ddls, ddl)
+				}
+			}
+		}
+	}
```

Comment:
It's fine to implement L210-215 here because your design of a default constraint is completely MSSQL-specific. However, I think we already implemented DROP of primary keys for other databases prior to column removal, so it seems weird that you need to take care of this here. Could you make either L143-152 or L155-168 take care of this, instead of individually duplicating the implementation only for MSSQL?